src/main/kotlin/ecommerce/model/Cart.kt

```diff
@@ -0,0 +1,33 @@
+package ecommerce.model
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.OneToMany
+import jakarta.persistence.OneToOne
+
+@Entity
+data class Cart(
+    @OneToOne(cascade = [CascadeType.PERSIST])
+    val member: Member,
+    @OneToMany(cascade = [CascadeType.ALL], mappedBy = "cart")
+    val cartProducts: MutableList<CartItem> = mutableListOf<CartItem>(),
```

Comment:
```suggestion
    val cartProducts: MutableList<CartItem> = mutableListOf(),
```

We can let this be implicit :)
Since this topic came up, it would be great to learn about generics if you have the time.
Generics in Kotlin could be quite hard, but it's used in logics very often.